learner/circuits.py

```diff
@@ -146,8 +148,9 @@ def layer(i, q):
                 Kgate(kappa[m, i]) | q[m]
         return q
 
-    # Start SF engine
-    eng, q = sf.Engine(2)
+    # Start SF program
+    sf.hbar = 2
```

Comment:
What is the motivation for this line? Did we also need it above, e.g., line 71

Reply:
Oops, good catch. yep, I noticed that the previous function used hbar=0.5 (can't remember why?), and so was meant to set `sf.hbar=0.5` there. Setting it back to 2 here was just as a precaution.